test/e2e/resources/stdlib/global-utilities.brs

```diff
@@ -1,3 +1,3 @@
 sub main()
-    print getInterface({}, "ifAssociativeArray");
```

Comment:
Any reason we don't want to test this as well?

Reply:
This isn't meant to be an exhaustive check of `getInterface`'s behavior, but testing with `{}` and `"ifAssociativeArray"` didn't force the automatic-promotion behavior that was added in this PR!